src/components/Button/index.tsx

```diff
@@ -82,11 +82,11 @@ export const Button = ({
         type === "primary"
           ? "bg-gradient-to-r from-origin-protocol-button-start to-origin-protocol-button-end"
           : "bg-transparent";
-      textColor = type === "primary" ? "text-white" : "text-black";
+      textColor = type === "primary" ? "text-white" : "text-slate-800";
```

Comment:
this would default all button styles to slate-800, is that what we want?

Reply:
is it necessary to use button in the header then, if it's just a link with no styling?

Reply:
we want some sort of hover indicator, and usually if there are dropdowns and buttons around a link it makes sense I think to keep it as a button with the additional padding for hover / click targets.

Reply:
Also there are definitely buttons on the story.xyz and op.com designs that use #000000 for button text, so this isn't the right way to do this. Maybe another ternary here that looks for the ousd webProperty if you need it?